tests/kafkatest/tests/produce_consume_validate.py

```diff
@@ -110,7 +111,8 @@ def validate(self):
         consumed = self.consumer.messages_consumed[1]
         missing = set(acked) - set(consumed)
 
-        self.logger.info("num consumed:  %d" % len(consumed))
+        self.logger.info("Producer produced %d messages where %d messages are unique" % (len(acked), len(acked)))
```

Comment:
I meant to use `len(set(acked))` for producer and `len(set(consumed))`. You are right that `len(set(acked))` doesn't make sense. Will fix it.